cypress/integration/plugins/reports-dashboards/04-download.spec.js

```diff
@@ -41,7 +41,7 @@ describe('Cypress', () => {
       .contains('PDF')
       .click({ force: true });
     cy.get('body').then(($body) => {
-      if ($body.find('#downloadInProgressLoadingModal').length > 0) {
+      if (cy.get('.euiToastHeader__title').contains('Successfully generated report').should('exist')) {
```

Comment:
don't know if this works, does `cy.get('.euiToastHeader__title').contains('Successfully generated report').should('exist')` return a boolean?

Reply:
i've update this code block to only look for toast message and also tested it locally.